curator/rh-manifest.txt

```diff
@@ -0,0 +1,19 @@
+boto3==1.12.33
+botocore==1.15.33
+certifi==2019.11.28
+chardet==3.0.4
+click==6.7
+docutils==0.15.2
+elasticsearch==7.0.4
+elasticsearch-curator==5.8.1
+futures==3.3.0
+idna==2.9
+jmespath==0.9.5
+python-dateutil==2.8.1
+PyYAML==3.13
+requests==2.23.0
+requests-aws4auth==0.9
+s3transfer==0.3.3
+six==1.14.0
+urllib3==1.24.2
+voluptuous==0.11.5
```

Comment:
Does the file format comply with https://mojo.redhat.com/docs/DOC-1195158?

Reply:
Per the doc there really is no specific file format, but received buy in from the person who asked me to add it